
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:25:48*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/auth/authenticate
Request: {"users":{"username":"shadymahmoud","user_password":"****"}}
@  2023-05-09T10:25:48
Error Name: Wrong credentials 
Message: Authenticatiion failed
Stack: Wrong credentials: Authenticatiion failed
    at E:\My_projects\RadAssist\Backend\dist\Authentication\Handlers\AuthenticationHandler.js:42:25
    at Generator.next (<anonymous>)
    at fulfilled (E:\My_projects\RadAssist\Backend\dist\Authentication\Handlers\AuthenticationHandler.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Data retrieved: {"result":null,"message":"username not found"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:26:02*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/auth/authenticate
Request: {"users":{"username":"shady mahmoud","user_password":"****"}}
@  2023-05-09T10:26:02
Error Name: Wrong credentials 
Message: Authenticatiion failed
Stack: Wrong credentials: Authenticatiion failed
    at E:\My_projects\RadAssist\Backend\dist\Authentication\Handlers\AuthenticationHandler.js:42:25
    at Generator.next (<anonymous>)
    at fulfilled (E:\My_projects\RadAssist\Backend\dist\Authentication\Handlers\AuthenticationHandler.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Data retrieved: {"result":null,"message":"username not found"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth OK*******************************************
*******************************@2023-05-09T10:26:14*******************************

Successful => Responded with status code: (200) - OK, Authenticated & Authorized
Request origin: 127.0.0.1/auth/authenticate
Request: {"users":{"username":"shady","user_password":"****"}}
@  2023-05-09T10:26:14
Data retrieved: {"action":"success","result":true,"accessToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InNoYWR5IiwiaWF0IjoxNjgzNjE3MTc0LCJleHAiOjE2ODM2MTg5NzR9.dM3A-iJDo7ciTF-rL216IaFgOqgmzy4HpPJCdV3fLOk","full_name":"shady mahmoud","username":"shady"}
*******************************************End Of Auth OK*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/users/index
Request: {"users":{"role":"radiologist","user_password":"****"}}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/users/index
Request: {"users":{"role":"radiologist","user_password":"****"}}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/studies/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/studies/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:346:12)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/patients/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:346:12)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/patients/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/users/index
Request: {"users":{"role":"radiologist","user_password":"****"}}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/users/index
Request: {"users":{"role":"radiologist","user_password":"****"}}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/studies/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/studies/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:346:12)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/patients/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JWT is null Error 
Message: JWT is null || not send in Authorization Header
Stack: JWT is null Error: JWT is null || not send in Authorization Header
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:54:29
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:346:12)
Data retrieved: {"message":"Authorization headers are not provided properly"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T10:51:25*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/patients/index
Request: {}
@  2023-05-09T10:51:25
Error Name: JsonWebTokenError 
Message: jwt must be provided
Stack: JsonWebTokenError: jwt must be provided
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:60:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:58:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:328:13)
    at E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\index.js:286:9
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth OK*******************************************
*******************************@2023-05-09T10:52:21*******************************

Successful => Responded with status code: (200) - OK, Authenticated & Authorized
Request origin: 127.0.0.1/auth/authenticate
Request: {"users":{"username":"123","user_password":"****"}}
@  2023-05-09T10:52:21
Data retrieved: {"action":"success","result":true,"accessToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEyMyIsImlhdCI6MTY4MzYxODc0MSwiZXhwIjoxNjgzNjIwNTQxfQ._zW_88AZ03cw2P4yzEvxQ8QZvOoGV7EcFh87iPT6h-o","full_name":"Test","username":"123"}
*******************************************End Of Auth OK*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T11:02:34*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/auth/refjwt
Request: {}
@  2023-05-09T11:02:34
Error Name: JsonWebTokenError 
Message: jwt malformed
Stack: JsonWebTokenError: jwt malformed
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:70:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:57:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth Failed*******************************************
*******************************@2023-05-09T11:02:34*******************************

UnAuthenticated => Responded with status code: (401) - unauthorized, Authetnication failed
Request origin: 127.0.0.1/auth/refjwt
Request: {}
@  2023-05-09T11:02:34
Error Name: JsonWebTokenError 
Message: jwt malformed
Stack: JsonWebTokenError: jwt malformed
    at module.exports [as verify] (E:\My_projects\RadAssist\Backend\node_modules\jsonwebtoken\verify.js:70:17)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:57:36
    at Generator.next (<anonymous>)
    at E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:4:12)
    at refreshAccessToken (E:\My_projects\RadAssist\Backend\dist\Authentication\MiddleWares\HandleToken.js:48:12)
    at Layer.handle [as handle_request] (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\My_projects\RadAssist\Backend\node_modules\express\lib\router\route.js:114:3)
Data retrieved: {"message":"Authentication Error"}
*******************************************End Of Auth Failed*******************************************

	
*******************************************Start Of Auth OK*******************************************
*******************************@2023-05-09T11:02:47*******************************

Successful => Responded with status code: (200) - OK, Authenticated & Authorized
Request origin: 127.0.0.1/auth/authenticate
Request: {"users":{"username":"123","user_password":"****"}}
@  2023-05-09T11:02:47
Data retrieved: {"action":"success","result":true,"accessToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEyMyIsImlhdCI6MTY4MzYxOTM2NywiZXhwIjoxNjgzNjIxMTY3fQ.QUAR66IpsnjLdn_hrMQG0vzrSD9XT49nohby9Wx2xOY","full_name":"Test","username":"123"}
*******************************************End Of Auth OK*******************************************

	